fast/stages/2-networking-a-peering/variables.tf

```diff
@@ -167,12 +173,15 @@ variable "psa_ranges" {
   # }
 }
 
-variable "region_trigram" {
-  description = "Short names for GCP regions."
-  type        = map(string)
+variable "regions" {
+  description = "Region definitions."
+  type = object({
+    primary   = string
+    secondary = string
```

Comment:
would it make sense to support more than one secondary?

Reply:
easy to add when/if needed